cli/root.go

```diff
@@ -34,6 +33,8 @@ var rootCmd = &cobra.Command{
 	syncs them to the given targets. This CLI is useful for
 	targets that do not support external credentials.
 	Support Jenkins only for now.`,
+	SilenceUsage:  true,
+	SilenceErrors: true,
```

Comment:
This makes it so when returning an error from a command, we don't:
1. Show the usage string
2. We tell the command we'll handle ourselves the errors coming from child commands